swagger-kondo.json

```diff
@@ -0,0 +1,17 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Samsara API",
```

Comment:
Nit: can we call this `Samsara API [PREVIEW]`?

Reply:
Changed.